doc/dune-files.rst

```diff
@@ -2,10 +2,195 @@
 Stanza Reference
 ****************
 
+.. _config:
+
+``config``
+==========
+
+This file is used to set global configuration of Dune (applicable across
+projects and workspaces).
+
+The configuration file is normally ``~/.config/dune/config`` on Unix systems and
+``%LOCALAPPDATA%/dune/config`` on Windows. However, it is possible to specify an
+alternative configuration file with the ``--config-file`` option.
+
+Command-line flags take precedence over the contents of the ``config`` file.
+
+If ``--no-config`` or ``-p`` is passed, Dune will not read this file.
+
+It can contain the following stanzas:
+
+.. _display:
+
+``display``
+-----------
+
+Specify the amount of Dune’s verbosity.
+
+.. code:: scheme
+
+    (display <setting>)
+
+where ``<setting>`` is one of:
+
+- ``progress``, Dune shows and updates a status line as build goals are being
+  completed (this is the default).
+
+- ``verbose`` prints the full command lines of programs being executed by Dune,
+  with some colors to help differentiate programs.
+
+- ``short`` prints one line per executed command, with the binary name on
+  the left and the reason it is being executed for on the right.
+
+- ``quiet`` only display errors.
+
+.. _concurrency:
+
+``concurrency``
+---------------
+
+Number of cores Dune is allowed to utilize.
```

Comment:
So "cores" might not be the correct thing to say here. Perhaps:
```
Maximum number of concurrent jobs Dune is allowed to have.
```

Reply:
Fixed (both).